variables.tf

```diff
@@ -850,6 +850,16 @@ variable "cloudsql_net_write_timeout" {
   default     = "240"
 }
 
+variable "cloudsql_db_user" {
+  description = "CloudSQL database user"
+  default     = "forseti_security_user"
+}
+
+variable "cloudsql_db_password" {
+  description = "CloudSQL database password"
+  default     = ""
+}
+
```

Comment:
Can you please add the same variables to [modules/on_gke/variables.tf](https://github.com/forseti-security/terraform-google-forseti/blob/master/modules/on_gke/variables.tf)?

Reply:
Updated, PTAL